app/code/community/Bolt/Boltpay/etc/config.xml

```diff
@@ -295,6 +295,14 @@
           <model>boltpay/cron::cleanupQuotes</model>
         </run>
       </immutable_quote_cleanup>
+      <preauth_order_cleanup>
+        <schedule>
+          <cron_expr>*/15 * * * *</cron_expr>
```

Comment:
Does this mean we are running every 15 minutes. If not what is the time frame and what is the reason we chose this interval.

Reply:
@mccottry I think 15mins might be too aggressive too. This is really for AKIRA where we want to stop Failed Webhooks and has a shitty server. Maybe we can do every hour?

Reply:
Oh I see I see, okay lets maybe keep it at 20mins to give us a little bit of a buffer

Reply:
bump

Reply:
20 minutes it is